### ISSUE TITLE
Feature request: exclude also i18n URLs

### ISSUE THREAD

**Issue by eriktelepovsky** [metalogico/django-sonar#9]
Hello. 

It would be great if ``DJANGO_SONAR['excludes']`` could automatically detect language code prefix in the URL path. So instead of setting this:

```python
DJANGO_SONAR = {
    'excludes': [
        '/en/jsi18n/', '/sk/jsi18n/', '/cs/jsi18n/', '/pl/jsi18n/', '/hu/jsi18n/', '/uk/jsi18n/',
    ],
}
```

it would be able to use a using single path instead:

```python
DJANGO_SONAR = {
    'excludes': [
        '/jsi18n/',
    ],
}
```

The jsi18n path is defined in urls.py using i18n patterns:

```python
urlpatterns += i18n_patterns(
    path('jsi18n/', JavaScriptCatalog.as_view(), name='javascript-catalog'),
)
```

Thanks!

**Comment by metalogico** [comment]
Hello again.

Thanks for all these suggestions man.

I think the best way to solve these kind of issues is to support regex on the "excludes" parameter.
In that way one could do something like this:

 /*/jsi18n/ 

I'll look into it for a future release.

**Comment by eriktelepovsky** [comment]
Yes, that's exactly what I meant :) Thank you! Looking forward to it.

**Comment by metalogico** [comment]
releases in 0.2.1
